### ISSUE TITLE
Unable to locate config file or config directory

### ISSUE THREAD

**Issue by nisrulz** [zzossig/hugo-theme-zzo#45]
Followed the steps in readme.
Deleted the config.toml file and created the config folder at the root. Added default as child to this config folder and then later added 4 config files.

When attempting to run `hugo server -D` I get the below error:
```
Unable to locate config file or config directory. Perhaps you need to create a new site.
       Run `hugo help new` for details.
```

**Comment by zzossig** [comment]
Can you try this quick solution?
Copy the `config`, `content`, `static` folder from the Zzo theme's exampleSite folder. And paste it to your root folder.

**Comment by nisrulz** [comment]
Hi, thank you for your reply.

I think I found the issue. It isn't very clear, but when you pointed me to the exampleSite folder I noticed there is an `_` infront of `default` folder. Checked readme and it is there too. So adding that enabled the build to go through.

![Screenshot from 2019-12-30 10-21-22](https://user-images.githubusercontent.com/2096087/71575723-6303d500-2aee-11ea-9aa8-a6d37e473919.png)

Maybe, this should be mentioned or highlighted in the readme. It is easy to miss if not copy pasted but done manually (which is what I did). 

Closing as this isn't an issue anymore.

Thank you for your support!
